content/authors/Dr.W4b4L/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+name: "Dr. W4b4L"
+images: []
+twitter: ""
+---
```

Comment:
Can you add some informations about DR. W4b4l here ?

Reply:
He want keep the anonymous